client/src/main/proto/spine/base/version.proto

```diff
@@ -19,26 +19,28 @@
 //
 syntax = "proto3";
 
-package spine.server.entity;
+package spine.base;
 
 option (type_url_prefix) = "type.spine3.org";
-option (SPI_all) = true;
-option java_package = "org.spine3.server.entity.status";
+option java_package = "org.spine3.base";
+option java_outer_classname = "VersionProto";
 option java_multiple_files = true;
-option java_outer_classname = "EntityStatusProto";
 option java_generate_equals_and_hash = true;
 
+import "google/protobuf/timestamp.proto";
+
 import "spine/annotations.proto";
 
-// The storage status of an entity.
-//
-// If one of the flags is set to `true` the entity becomes “invisible” to regular queries.
-//
-message EntityStatus {
+// A version number with timestamp of the version creation.
+message Version {
 
-    // If `true` the aggregate will not be loaded for regular queries.
-    bool archived = 1;
+    // A zero-based version number.
+    //
+    // When numbering versions of entities the zero value is reserved for
+    // the “pre-initialization” version of an entity.
+    //
+    int32 number = 1;
```

Comment:
Well spotted. The initial intent was to use it in the `CommandContext`, which doesn't have to have the target version. But it keeps `int32` now. So, I'll make both `Version` fields required, and we'll see. We can relax it back should we see a need.